docs/node/clients/python_wrapper.md

```diff
@@ -0,0 +1,931 @@
+# Python-Wrapper Documentation:
+
+>Important Preface:  
+All of the documentation and testing done regarding the VILLASnode Python-Wrapper
+is based upon the signal_v2 node provided by VILLASnode.
+Node specific functions are implemented on a node to node basis and 
+therefore may exert different behavior.
+
+1. [VILLASnode functions exposed by the C-API](#1.-villasnode-functions-exposed-by-the-c-api)
+    - [Functions to set up a node or modify its state](#functions-to-set-up-a-node-or-modify-its-state)
+    - [Functions to extract node specific information](#functions-to-extract-node-specific-information)
+    - [Functions related to data transfer](#functions-related-to-data-transfer)
+2. [Using the C-API with the Python-Wrapper](#2.-using-the-c-api-with-the-python-wrapper)
+3. [Bugs](#3.-bugs)
+4. [Improvements](#4.-improvements)
+5. [Contributing to the Python-Wrapper](#5.-contributing-to-the-python-wrapper)
+6. [Installation](#6.-installation)
+
+## 1. VILLASnode functions exposed by the C-API
```

Comment:
Can we link to the actual header from the VILLASnode repo here?

E.g: https://github.com/VILLASframework/node/blob/master/include/villas/node.h